src/Lumi/Components/DropdownButton.purs

```diff
@@ -249,7 +251,8 @@ styles = jss
   { "@global":
       { "lumi-dropdown-button":
           { display: "inline-block"
-          , "& > button.lumi":
+          , position: "relative"
```

Comment:
Do we need this?

Reply:
Yeah, without it the right-aligned dropdown isn't positioned correctly. I'm not really sure why it worked before to be honest :sweat_smile:

Reply:
I'm going to ask Arthur to review as well actually, he might be more familiar with this part.